kotlin/internal/jvm/android.bzl

```diff
@@ -63,19 +64,20 @@ def _kt_android_artifact(
             **kwargs
         )
         exported_target_labels.append(base_name)
-    elif resource_files or manifest:
+    elif resource_files or manifest or assets:
         # TODO(https://github.com/bazelbuild/rules_kotlin/issues/556): replace with starlark
         # buildifier: disable=native-android
         # Do not export deps to avoid all upstream targets to be invalidated when ABI changes.
+        kwargs.pop("exports", default = None)
         _android_library(
             name = base_name,
             resource_files = resource_files,
             deps = deps,
             custom_package = kwargs.get("custom_package", default = None),
-            manifest = manifest,
```

Comment:
This removal is here because, if we are passing `kwargs`, `manifest` will still be in the dict (by virtue of using `.get`), and so will be passed twice.